CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.4.3)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
What feature(s) are used that don't work in 2.4.3? Just bumping up to 3.0 without reason doesn't make sense.